src/manage_license.h

```diff
@@ -0,0 +1,91 @@
+/* Copyright (C) 2020-2021 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @file manage_license.c
+ * @brief GVM management layer: License information headers.
+ *
+ * Headers for non-SQL license information code for the GVM management layer.
+ */
+
+#include <glib.h>
+
+/* Data types */
+
+/**
+ * @brief Defines the metadata of a license
+ */
+typedef struct {
+  char *id;               ///< Unique Identifier of the license
+  int  schema_version;    ///< Version of the license file schema
```

Comment:
IMHO besides using version only this should be a `char *` too.
```suggestion
  int  version;    ///< Version of the license file schema
```